CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+  
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## LibJava-Skycoin 0.25.1 - 2019/03/19
```

Comment:
Shouldn't it be Android instead of Java ?

Reply:
Yes

Reply:
Done in last commit